app/src/main/java/de/tum/in/tumcampusapp/component/ui/transportation/MVVSymbolView.java

```diff
@@ -67,8 +73,10 @@ public MVVSymbolView(String line) {
                 } else if (num == 8) {
                     triangle = 2;
                     backgroundColor = 0xffeb6a27;
-                } else {
+                } else if (num != 0) {
                     backgroundColor = U_LINE_COLOR[num - 1];
```

Comment:
:man_facepalming: 
we already had the `Optional.fromNullable(...).or(0)` in place, but then accessed out of bounds